mathics/core/atoms.py

```diff
@@ -942,6 +942,8 @@ def is_zero(self) -> bool:
 
 
 RationalOneHalf = Rational(1, 2)
+MATHICS_COMPLEX_I = Complex(Integer0, Integer1)
```

Comment:
We have been moving from Mathics to Mathics3, and this is a Mathics3-only change. Consider whether we want to use MATHICS or MATHICS3.